.github/workflows/build_and_test.yml

```diff
@@ -94,23 +94,38 @@ jobs:
       run: sudo ./libafl_concolic/test/smoke_test_ubuntu_deps.sh 
     - name: Run smoke test
       run: ./libafl_concolic/test/smoke_test.sh 
-  ubuntu-fuzzers:
-    runs-on: ubuntu-latest
+  fuzzers:
+    env:
+      CC: ccache clang # use ccache in default
+    strategy:
+      matrix:
+          os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
     steps:
     - uses: actions-rs/toolchain@v1
       with:
         profile: minimal
         toolchain: stable
+    - name: add ccache 
+      uses: hendrikmuhs/ccache-action@v1.2
+      with:
+        key: ${{ github.job }}-${{ matrix.os }}
+        max-size: 2000M
```

Comment:
Maybe we want to increase the size?

Reply:
Previous run shows that it only takes about ~200M, so `max-size: 2000M` is enough.
The lastest run add ccache to `CXX`, wait for the result to see if we should adjust the size.